src/hevm/src/EVM/SymExec.hs

```diff
@@ -414,12 +412,26 @@ simplify e = if (mapExpr go e == e)
       | a == b = (And b c)
       | otherwise = o
 
+    -- go (And (Lit x) b)
+    --   | x == 0 = Lit 0
+    --   | otherwise = b
+    -- go (And a (Lit x))
+    --   | x == 0 = Lit 0
+    --   | otherwise = a
```

Comment:
Can we remove this commented code?

Reply:
Hmmm I fixed this yesterday, maybe I forgot to push? It's fixed now :)